src/Component/Symbolizer/TextEditor/TextEditor.tsx

```diff
@@ -215,7 +215,7 @@ export class TextEditor extends React.Component<TextEditorProps> {
           <span className="label">{locale.templateFieldLabel}:</span>
           <Mention
             placeholder={locale.templateFieldLabel}
-            defaultValue={toContentState(symbolizer.label)}
+            // defaultValue={toContentState(symbolizer.label)}
```

Comment:
Is this really not needed any more?

Reply:
This was commited acciedently. I'll revert the changes in `TextEditor`.